site/layouts/partials/header.html

```diff
@@ -2,6 +2,7 @@
 <html lang="{{ $.Site.Language.Lang }}">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Heads up for [Designing Websites for iPhone X](https://webkit.org/blog/7929/designing-websites-for-iphone-x/) iOS 11 introduces a new extension for the existing `viewport` meta tag called [`viewport-fit`](https://www.w3.org/TR/css-round-display-1/#viewport-fit-descriptor), which provides control over the insetting behavior. The default setting is `auto`, which will not cover the entire screen (a long-standing issue in iOS splash screen and full-screen hero design).

In order to disable the default inset behavior and cause the page to lay out to the full size of the screen, you can set `viewport-fit` to `cover` as shown here:

    <meta name='viewport' content='initial-scale=1, viewport-fit=cover'>

Thanks for taking the torch on this repo @bdougie. You da man!

Reply:
I will fix this shortly :) Thank you for the hint.